src/Sylius/Bundle/CoreBundle/DependencyInjection/SyliusCoreExtension.php

```diff
@@ -58,7 +58,7 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('sylius_core.catalog_promotions.batch_size', $config['catalog_promotions']['batch_size']);
 
         $env = $container->getParameter('kernel.environment');
-        if ('test' === $env || 'test_cached' === $env) {
+        if (str_starts_with($env, 'test')) {
```

Comment:
This change doesn't mean the same as it was before, it's way more generic.

Reply:
that's the whole point - our use case is as follows:
we are modularizing Sylius Plus. In order to ensure its interoperability among other libs, we have to test them sometimes with other plugins. This influences configuration in some cases. So we can either have a script that will adjust the application to these changes or use a separate test env. Separate test env seems to be a better way. To be exact:
We have MultiStorePlugin and it has `test` env for testing purposes of it is own. It has additional `test_invoicing` env for testing MultiStorePlugin with InvoicingPlugin (because there are small adjustments between them two). This change will allow me to use Behat services without additional include :)

Reply:
IMHO it should be marked as BC change, even if it's a small one.

Reply:
why do you consider it a Backward Compatibility change? How can I break someone's app? The only thing, that will change is to add these services: https://github.com/Sylius/Sylius/blob/1.12/src/Sylius/Bundle/CoreBundle/Resources/config/test_services.xml

Perhaps, I'm too optimistic, but I didn't see the usage of custom envs too often. I'm still expecting to have `test` keyword at the beginning of env.

Also, I don't see any rule broken according to our BC-Policy: https://docs.sylius.com/en/latest/book/organization/backward-compatibility-promise.html